aries-site/src/pages/components/tabs.mdx

```diff
@@ -39,9 +40,31 @@ the browser location.** This will allow proper active styling and routing to fun
 ### Alignment
 
 To maintain a clean, consistent presentation, all content should be aligned to the left edge of the Tabs component.
+
 - Apply `justify="start"` to left-align the tabs and have their bottom border extend the full-width of the parent container.
 - Content within the Tab should be flush with the left edge of the Tabs component and have a vertical padding of `medium` to provide spacing and room for the content to breathe.
 
+### Responsive Tabs
+
+When horizontal space is limited, only the tabs that fit will be displayed. The remaining tabs can be accessed by using the previous and next arrows or with the keyboard.
+
+#### Responsive Behavior
+
+- Previous and next arrows will be present if the horizontal tabs are fully outside the viewport.
+- Previous and next arrows are disabled if the horizontal tab has no more tabs to view towards the left or right.
```

Comment:
am I missing something are we not capitalizing components anymore ? Tabs ?

Reply:
Oops yeah these should be capitalized. Fixed this in latest commit